content/brazilian-portuguese/arduino/activity-4.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Atividade 4 - Display LCD"
+draft: false
+weight: 7
+---
+
+## Diagrama de fiação
+
+Agora que você domina os sinais de saída para LEDs, vamos ver a exibição de texto com uma exibição simples. Apenas 1 resistor (5K) é usado para limitar o brilho do display. Tente ligar isso de acordo com o diagrama sem instruções específicas.
+
+![Alt Text: Diagrama de fiação para Elegoo e LCD Display](../img/LCD-wiring.png)
+
+## O Código:
+``` c++
+//www.elegoo.com
+//2016.12.9
+
+/*
+  Biblioteca LiquidCrystal - Hello World
+
+Demonstra o uso de uma tela LCD 16x2.  O Cristal Líquido
+ biblioteca funciona com todos os monitores LCD que são compatíveis com o
+ Hitachi HD44780 driver. Há muitos deles por aí, e você
+ geralmente pode dizê-los pela interface de 16 pinos.
+
+Este esboço imprime "Hello World!" no LCD
+ e mostra o tempo.
+
+O circuito:
+ * Pino RS LCD para pino digital 7
+ * LCD Ativar pino para pino digital 8
+ * LCD D4 pino para pino digital 9
```

Comment:
Reorder the words. Suggestion: "Pino D4 do LCD para pino digital 9". Do the same in lines 33, to 38.